.github/workflows/test_pinned_deps.yml

```diff
@@ -51,7 +51,7 @@ jobs:
       - name: Pytest with coverage
         if: matrix.os == 'ubuntu-latest'
         # Note: Use of pytest -n .. (pytest-xdist) does not work with coverage
-        run: coverage run --source=src/paste3 -m pytest
+        run: coverage run --source=src/paste3 --omit="*/__main__.py" -m pytest
```

Comment:
can we just add a test to check the execution of the `main` function in `__main__.py` (or even just if `{sys.executable} paste3 --version` will work? In any case, let's not make exceptions like this (let coverage go down if it must).

Reply:
Added a test to check version number and got rid of the `--omit...`.